qualtran/bloqs/chemistry/trotter/trotterized_unitary.py

```diff
@@ -102,9 +102,11 @@ def __attrs_post_init__(self):
     def signature(self) -> Signature:
         return self.bloqs[0].signature
 
-    def build_composite_bloq(self, bb: 'BloqBuilder', **soqs: SoquetT) -> Dict[str, 'Soquet']:
+    def build_composite_bloq(self, bb: 'BloqBuilder', **soqs: SoquetT) -> Dict[str, 'SoquetT']:
         for i, a in zip(self.indices, self.coeffs):
-            soqs |= bb.add_d(attrs.evolve(self.bloqs[i], angle=2 * a * self.timestep), **soqs)
+            # Bloqs passed in are supposed to be attrs dataclasses per docs
+            # It would be nice to somehow specify that self.bloqs are both bloqs and AttrsInstance
```

Comment:
is it worth opening an issue for this?

Reply:
No, I think it is blocked on features needed for mypy which look like it's not happening.
I will leave this comment in there in case anyone who is a type-guru wants to fix it, but otherwise not worth it, in my opinion.